packages/cli-e2e/utils/platform.ts

```diff
@@ -0,0 +1,8 @@
+import {HTTPRequest} from 'puppeteer-core';
+
+export function isSearchRequest(request: HTTPRequest) {
```

Comment:
This file is a bit weird as it's the only function in there.

How about we have just one "test utils" file, and we split it when we see that it start becoming bloated with a ton of stuff ?

Having a bunch of different utils file like that causes other types of problems, where it's hard to know what already exists or not.

Reply:
Maybe we can move it in the browser utils, so we can have 3 utils (for now):
* cli
* browser
* login (or auth)